feature-detects/messagechannel.js

```diff
@@ -0,0 +1,22 @@
+/*!
+{
+  "name": "Message Channel",
+  "property": "MessageChannel",
+  "authors": ["Raju Konga [kongaraju]"],
+  "caniuse" : "MessageChannel",
+  "tags": ["performance", "messagechannel"],
+  "notes": [{
+    "name": "W3C Reference",
+    "href": "https://www.w3.org/TR/2011/WD-webmessaging-20110317/#message-channels"
+  }, {
+    "name": "MDN documentation",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API/Using_channel_messaging"
+  }]
+}
+!*/
+/* DOC
+Detects support for  `Message Channel` API from the Channel Messaging spec. Message Channels  provide a simple way for communication between threads.
```

Comment:
There is weird spacing, its a bit overly verbose, and not 100% accurate.

Could you update to something like

```
Detects support for Message Channels, a way to communicate between different browsing contexts (iframes, workers, etc)
```